db/seeds.rb

```diff
@@ -15,8 +15,8 @@
   {award_name: 'Outstanding Youth Tech Award', relevant_fields: 1}])
 
 users = User.create([
-  {email: 'tommy@aol.com', password: 'hello314', first_name: 'Tommy', last_name: 'Tutone', weight: 3, admin: true},
-  {email: 'petericketts@netscape.com', password: 'isuckl0ts', first_name: 'Pete', last_name: 'Ricketts', weight: 1, admin: false}])
+  {email: 'tommy@aol.com', password: 'hello314', full_name: 'Tommy Tutone', weight: 3, admin: true},
```

Comment:
Use 2 spaces for indentation in an array, relative to the first position after the preceding left parenthesis.